packages/accordion/README.md

```diff
@@ -23,41 +20,44 @@
 </vaadin-accordion>
 ```
 
-[<img src="https://raw.githubusercontent.com/vaadin/vaadin-accordion/master/screenshot.png" alt="Screenshot of vaadin-accordion" width="900">](https://vaadin.com/components/vaadin-accordion)
+[<img src="https://raw.githubusercontent.com/vaadin/web-components/master/packages/accordion/screenshot.png" alt="Screenshot of vaadin-accordion" width="900">](https://vaadin.com/docs/latest/ds/components/accordion)
 
 ## Installation
 
-Install `vaadin-accordion`:
+Install the component:
 
 ```sh
 npm i @vaadin/accordion --save
```

Comment:
`--save` should not be necessary 🤔

Reply:
Updated. Will do the same for all the `vaadin-` packages in a separate PR.